resources/public/css/main.css

```diff
@@ -0,0 +1,17 @@
+blue-border {
```

Comment:
This doesn't refer to any actual element in the page, it would only if we had `<blue-border>` tags. Either do classes, ids, or take that out. Same goes for `main-text` below.